### PR TITLE
[ZT] Access for SaaS SCIM provisioning

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/applications/configure-apps/saas-apps/_index.md [hunk 1]
```diff
@@ -8,4 +8,76 @@ weight: 1
...
 
+## Add a SaaS application
+
+Cloudflare integrates with the majority of SaaS applications that support the SAML or OIDC authentication protocol. If you do not see your application listed below, refer to our generic SAML or generic OIDC guide and consult your SaaS application's documentation.
+
 {{<directory-listing>}}
+
+## Configure SCIM
+
+[SCIM provisioning](/cloudflare-one/identity/users/scim/) allows you to synchronize users and groups between your identity provider (IdP) and SaaS app. Once configured, SCIM will automatically grant a user access to all the SaaS applications they need to do their job once they have been added to the IdP. Each SaaS application has different SCIM setup requirements, and some applications may not support all of the SCIM functionality available in Zero Trust.
```
Comment (ranbel) [inline content/cloudflare-one/applications/configure-apps/saas-apps/_index.md:19]:
```suggestion
[SCIM provisioning](/cloudflare-one/identity/users/scim/) synchronizes users and groups between your identity provider (IdP) and SaaS app. Once configured, SCIM will automatically grant a user access to all the SaaS applications they need to do their job once they have been added to the IdP. Each SaaS application has different SCIM setup requirements, and some applications may not support all of the SCIM functionality available in Zero Trust.
```

File: content/cloudflare-one/applications/configure-apps/saas-apps/_index.md [hunk 2]
```diff
@@ -8,4 +8,76 @@ weight: 1
...
+3. In the **Authentication** tab, turn on **SCIM provisioning**.
+4. Select your IdP.
+5. In **Remote URL**, enter the SCIM server URL obtained from your SaaS application account.
+6. Select the credential type compatible with your SaaS application.
+7. Enter the authentication credentials obtained from your SaaS application account.
+8. Select **Verify credentials**.
+9. (Optional) Configure [advanced provisioning settings](#advanced-provisioning-settings).
+10. Select **Save application**.
+
+Cloudflare will begin synchronizing users and groups between your IdP and the SaaS application according to the provisioning schedule set in your IdP. To track the user's identity over time, view your [User Registry audit logs](/cloudflare-one/insights/logs/users/).
```
Comment (ranbel) [inline content/cloudflare-one/applications/configure-apps/saas-apps/_index.md]:
```suggestion
Cloudflare will begin synchronizing users and groups between your IdP and the SaaS application according to the provisioning schedule set in your IdP. To track how the user's identity changes over time, view your [User Registry audit logs](/cloudflare-one/insights/logs/users/).
```

File: content/cloudflare-one/applications/configure-apps/saas-apps/_index.md [hunk 3]
```diff
@@ -8,4 +8,76 @@ weight: 1
 
 Cloudflare Access allows you to add an additional authentication layer to your SaaS applications. When you integrate a SaaS application with Access, users log in to the application with Cloudflare as the Single Sign-On provider. The user is then redirected to the configured identity providers for that application and are only granted access if they pass your Access policies.
 
+## Add a SaaS application
+
+Cloudflare integrates with the majority of SaaS applications that support the SAML or OIDC authentication protocol. If you do not see your application listed below, refer to our generic SAML or generic OIDC guide and consult your SaaS application's documentation.
+
 {{<directory-listing>}}
+
+## Configure SCIM
```
Comment (kennyj42) [inline content/cloudflare-one/applications/configure-apps/saas-apps/_index.md]: should we add a call out that this is a beta feature?

**Review by kennyj42 — APPROVED** [review]

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR. All PRs prior to August 12 need to be reworked due to the platform migration to Astro.

**Pull request closed without merging**